lib/qesapdeployment.pm

```diff
@@ -150,11 +150,12 @@ sub qesap_create_ansible_section {
 
 =head3 qesap_pip_install
 
-  Install all Python requirements of the qe-sap-deployment
+  Install all Python requirements of the qe-sap-deployment in a dedicated virtual environment
 =cut
 
 sub qesap_pip_install {
     zypper_call('in python39');
+    assert_script_run("python3.9 -m venv exec_env && source exec_env/bin/activate");
```

Comment:
I propose to use some 'global' reference to calculate the path for the venv. I'm talking about `exec_env` . I'm thinking about something private in the lib. Maybe something like

```
use constant QESAPDEPLOY_VENV => '/tmp/exec_venv';
```

Then code like

```
assert_script_run("python3.9 -m venv exec_env && source exec_env/bin/activate");
```

could be changed to

```
assert_script_run("python3.9 -m venv " . QESAPDEPLOY_VENV . " && source " . QESAPDEPLOY_VENV . "/bin/activate");
```

Reply:
Agreed, done and new vr attached